iterating-over-data/iterating-over-dictionaries.md

```diff
@@ -0,0 +1,321 @@
+# Iterating Over Dictionaries
+
+## Learning Goals
+
+- Practice using for loops to iterate through a dictionary
+
+## Introduction
+
+The overall concept for iterating over dictionaries is the same as iterating over lists. We can use `for` loops to iterate over **each key-value pair in a dictionary.**
+
+However, because dictionaries are key-value pairs, there are a few differences to pay attention to.
```

Comment:
```suggestion
The overall concept for iterating over dictionaries is the same as iterating over lists. We use `for` loops to iterate over each dictionary item.

However, because dictionary items are key-value pairs, there are a few ways that we can iterate over them: **by key-value pair**, **by key alone**, or **by value alone**.
```

Gonna see how it feels to drop your very nice transition sentence.